common/config/azure-pipelines/templates/core-build.yaml

```diff
@@ -84,18 +84,28 @@ steps:
       RUSH_BUILD_CACHE_ENABLED: ${{parameters.rushBuildCacheEnabled}}
       VITE_CI: true
 
+  - script: npm run android:all
+    workingDirectory: test-apps/display-test-app
+    displayName: Build & run Android display-test-app
+    env:
+      IMJS_OIDC_CLIENT_ID: $(IMJS_OIDC_CLIENT_ID)
+      IMJS_OIDC_CLIENT_SECRET: $(IMJS_OIDC_CLIENT_SECRET)
+      AZURE_DEVOPS_EXT_PAT: $(System.AccessToken)
+      TOKEN: $(GitHubPAT)
```

Comment:
what is this used for?

Reply:
The first two are the client ID and client secret needed for service authorization to download the test iModel. The Azure PAT is needed to download the Android upacks. The GitHubPAT is needed to download the mobile-native-android library.